### PR TITLE
Add F5 prefix to configuration opening paragraphs, style consistency

### PR DESCRIPTION
**Pull request by ADubhlaoich**
This commit changes each of the opening paragraphs of the Configuration section to ensure they have the F5 prefixed name for the first instance of NGINX Ingress Controller.

It also includes a large amount of other changes:

- Relative links
- Erroneous product names ("the Ingress Controller")
- Contemporary table formatting
- Updated and decoupled input/output code blocks
- Sentence case for page titles and headings
- Small contextual rewrites
- Updated links to appropriate tags

A small number of unrelated changes were also included, regarding NGINX App Protect and a small typo in a command line option.
Preview: https://deploy-preview-5737--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/

### Proposed changes

Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to
that issue here in this description (not in the title of the PR).

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [x] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Comment by ADubhlaoich** [comment]
This is the first of a series of PRs that will all address the same types of documentation consistency problems.

I decided I would split this effort into a section by section basis, so that I do not get mangled by merge issues.

File: docs/content/configuration/global-configuration/configmap-resource.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
 - ''
-title: ConfigMap Resources
+title: ConfigMap resources
 toc: true
-weight: 1600
+weight: 300
 ---
 
-The ConfigMap Resources allows you to customize or fine tune NGINX behavior. Examples include setting the number of worker processes or customizing the access log format.
+When using F5 NGINX Ingress Controller, you can customize or fine tune NGINX behaviour using ConfigMap resources. Examples include setting the number of worker processes or customizing the access log format.
```
Comment (j1m-ryan) [inline docs/content/configuration/global-configuration/configmap-resource.md:10]: > customize (US)
> behaviour (UK)
Is there a preferred style between US and UK English for our docs, or are both acceptable?

File: docs/content/configuration/handling-host-and-listener-collisions.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 docs: DOCS-590
 doctypes:
 - ''
-title: Handling Host and Listener Collisions
+title: Host and Listener collisions
```
Comment (vepatel) [inline docs/content/configuration/handling-host-and-listener-collisions.md:5]: I think we should keep the original heading, new on comes off a bit negative in the context

**Review by j1m-ryan — APPROVED** [review]

File: docs/content/configuration/global-configuration/configmap-resource.md (hunk 1 above)
Comment (ADubhlaoich) [reply docs/content/configuration/global-configuration/configmap-resource.md:10]: Good catch, we're meant to stick to US spelling. Updating it in a moment.

File: docs/content/configuration/handling-host-and-listener-collisions.md (hunk 2 above)
Comment (ADubhlaoich) [reply docs/content/configuration/handling-host-and-listener-collisions.md:5]: I didn't realise I hadn't done a full sweep of this doc: updating it now.

I stand by the page title, but I think this is more of a reference page than a how-to, as it largely explains the mechanisms of how it works: it's not particularly instructional, so I don't think it makes sense under "Configuration".

Comment (vepatel) [reply docs/content/configuration/handling-host-and-listener-collisions.md:5]: okay in that case we should rename the md file too for consistency

**Review by vepatel — APPROVED** [review]

File: docs/content/configuration/security.md [hunk 3]
```diff
@@ -17,13 +17,10 @@ For general guidance, we recommend the official Kubernetes documentation for [Se
...
-The Ingress Controller requires a service account which is configured using RBAC.
-We strongly recommend using the [RBAC configuration](https://github.com/nginxinc/kubernetes-ingress/blob/v3.5.2/deployments/rbac/rbac.yaml) provided in our standard deployment configuration. It is configured with the least amount of privilege required for the Ingress Controller to work.
+NGINX Ingress Controller requires RBAC to configure a [ServiceUser](https://kubernetes.io/docs/concepts/security/service-accounts/#default-service-accounts), and provides least privilege access in its standard deployment configurations:
 
-We strongly recommend inspecting the RBAC configuration for [Manifests](https://github.com/nginxinc/kubernetes-ingress/blob/v3.5.2/deployments/rbac/rbac.yaml)
-or for [Helm](https://github.com/nginxinc/kubernetes-ingress/blob/v3.5.2/charts/nginx-ingress/templates/clusterrole.yaml) to understand what access the Ingress Controller service account has and to which resources. For example, by default the service account has access to all Secret resources in the cluster.
+- [Helm](https://github.com/nginxinc/kubernetes-ingress/blob/v3.5.0/deployments/rbac/rbac.yaml)
```
Comment (vepatel) [inline docs/content/configuration/security.md:22]:
```suggestion
- [Helm](https://github.com/nginxinc/kubernetes-ingress/blob/v3.5.2/deployments/rbac/rbac.yaml)
```

File: docs/content/configuration/security.md [hunk 4]
```diff
@@ -17,13 +17,10 @@ For general guidance, we recommend the official Kubernetes documentation for [Se
...
+- [Manifests](https://github.com/nginxinc/kubernetes-ingress/blob/v3.5.0/deployments/rbac/rbac.yaml)
```
Comment (vepatel) [inline docs/content/configuration/security.md:23]:
```suggestion
- [Manifests](https://github.com/nginxinc/kubernetes-ingress/blob/v3.5.2/deployments/rbac/rbac.yaml)
```

**Review by vepatel — APPROVED** [review]

**Review by vepatel — APPROVED** [review]

**Pull request merged**